package.json

```diff
@@ -53,7 +53,8 @@
     "test:component:open:win:firefox": "cypress open --component --browser firefox",
     "test:component:open:win:chrome": "cypress open --component --browser chrome",
     "test:component:open:win:edge": "cypress open --component --browser edge",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "create:component": "npx tsc src/utils/create_component_file/index.ts && node src/utils/create_component_file/index.js"
```

Comment:
`&&` is multi platform behavior?

Reply:
Yes, this works in [cmd shell](https://ss64.com/nt/syntax-conditional.html) and in [Unix bash and zsh](https://www.gnu.org/software/bash/manual/bash.html#Definitions).